charts/chronograf/values.yaml

```diff
@@ -131,3 +131,10 @@ envFromSecret: ""
 # - name: chronograf-output-influxdb2
 #   mountPath: /var/lib/chronograf/conf.d
 #   subPath: influxdb2.conf
+
+influxdb:
+  ## Name of an existing Secrect used to set the environment variables for the
+  ## InfluxDB user and password. The expected keys in the secret are
+  ## `influxdb-user` and `influxdb-password`.
+  ##
+  # existingSecret: influxdb-auth
```

Comment:
I think calling just `existingSecret` as in the kapacitor would be confusing.

https://github.com/influxdata/helm-charts/blob/20aeb75f75ecf739c8134edec7c5e8ff249d85f5/charts/kapacitor/values.yaml#L68